example/calculate_pi/pi_driver.cpp

```diff
@@ -111,7 +111,7 @@ void PiDriver::PostExecute(Real pi_val) {
     fs << "rel error = " << (pi_val - M_PI) / M_PI << std::endl;
     fs.close();
   }
-  Driver::PostExecute();
+  Driver::PostExecute(DriverStatus::complete);
```

Comment:
Just making sure this should be part of this PR and isn't meant for another PR

Reply:
I snuck this in here as an issue that keeps coming up but never with enough importance to generate a PR.